code/FulpstationCode/fulp_emag/fulp_emag.dm

```diff
@@ -55,7 +58,23 @@
 
 	if(istype(A, /mob/living/silicon/robot))
 		var/mob/living/silicon/robot/R = A
-		if(!R.emagged)
-			return TRUE
+		if(pre_check && (R.opened || R.locked) ) //Checks specific to pre emag act.

```

Comment:
shouldn't emagging a borg with a closed and locked cover to unlock its cover cost an emag charge?